Formula/php.rb

```diff
@@ -291,6 +295,11 @@ def plist; <<~EOS
 
   test do
     assert_match /^Zend OPcache$/, shell_output("#{bin}/php -i"), "Zend OPCache extension not loaded"
+    # Test related to libxml2 and
+    # https://github.com/Homebrew/homebrew-core/issues/28398
+    assert_match /^\s#{Formula["libpq"].opt_prefix}/,
+      shell_output("otool -L #{bin}/php"),
```

Comment:
Use `MachO::Tools.dylibs(app)` rather than `otool`. There's an example in `corsixth` if you want one.